lib/ansible/modules/web_infrastructure/wildfly.py

```diff
@@ -0,0 +1,326 @@
+ANSIBLE_METADATA = {
+  'status': ['preview'],
+  'supported_by': 'community',
+  'metadata_version': '1.0'
+}
+
+DOCUMENTATION = """
+module: wildfly
+version_added: "2.4"
+short_description: Deploy applications to Wildfly.
+description:
+  - Deploy applications to Wildfly domain and standalone mode.
+options:
+  deployment:
+    required: true
+    description:
+      - The name of the deployment.
+  src:
+    required: false
+    description:
+      - The remote path of the application ear or war to deploy.
+  controller_address:
+    required: true
+    default: http://localhost:9990
+    description:
+      - The HTTP API management address.
+  state:
+    required: false
+    choices: [ present, absent ]
+    default: present
+    description:
+      - Whether the application should be deployed or undeployed.
+  username:
+    required: true
+    description:
+      - The HTTP API management username.
+  password:
+    required: true
+    description:
+      - The HTTP API management password.
+  mode:
+    required: false
+    default: domain
+    description: Domain mode or standalone mode
+  server_groups:
+    required: false
+    default: all-relevant-server-groups
+    description: Server groups to deploy the war.
+author: "Aditya Cheelangi Sreedharamurthy"
+"""
+
+EXAMPLES = """
+# Deploy a hello world application
+- name: Deploy an application in domain mode
+  wildfly:
+    src: /tmp/hello-1.0-SNAPSHOT.war
+    deployment: hello.war
+    state: present
+    controller_address: http://localhost:9990
+    username: username
+    password: password
+    mode: domain
+    server_groups: all-relevant-server-groups
+# Undeploy the hello world application
+- name: Undeploy an application in domain mode
+  wildfly:
+    deployment: hello.war
+    state: absent
+    controller_address: http://localhost:9990
+    username: username
+    password: password
+    mode: domain
+    server_groups: all-relevant-server-groups
+# Deploy in standalone mode
+- name: Deploy an application in standalone mode
+  wildfly:
+    src: /tmp/hello-1.0-SNAPSHOT.war
+    deployment: hello.war
+    state: present
+    controller_address: http://localhost:9990
+    username: username
+    password: password
+    mode: standalone
+"""
+
+RETURN = '''
+---
+state:
+  description: State of the deployment.
+  returned: success
+  type: string
+  sample: deployed
+name:
+  description: Name of the deployed content.
+  returned: success
+  type: string
+  sample: hello.war
+'''
+
+HAS_JSON = True
+
+try:
+    import json
+except ImportError:
+    try:
+        import simplejson as json
+    except ImportError:
+        HAS_JSON = False
+
+import ansible.module_utils.six.moves.urllib.error as urllib_error
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.urls import *
+from ansible.module_utils.pycompat24 import get_exception
+
+
+def undeploy(params):
+    deployment = params['deployment']
+    remove_content = \
```

Comment:
Format this like this:

```
remove_content = {
    'operation': 'remove',
    'address': [
        {'deployment': deployment}
    ],
    'json.pretty': 1
}
```

The same anywhere else where you are using ` = \`.